pynicotine/shares.py

```diff
@@ -1055,7 +1055,8 @@ def addToShared(self, name):
         sharedstreams = config["transfers"]["sharedfilesstreams"]
         wordindex = config["transfers"]["wordindex"]
         fileindex = config["transfers"]["fileindex"]
-        shareddirs = config["transfers"]["shared"] + [config["transfers"]["downloaddir"]]
+        shareddirs = [path for _name, path in config["transfers"]["shared"]]
+        sharedmtimes.append(config["transfers"]["downloaddir"])
```

Comment:
Appending to `sharedmtimes` before defining it in the line below will not work - I don't know how that worked for you? I think you meant `shareddirs`.

IMHO also it's best to take `sharedmtimes` out completely, because it's not used. I did that in #59, while keeping your fix for `shareddirs`.